docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,148 @@ Zonemaster
+You can also get further insight about each test from the [Implemented Test Cases] document.
 
 <a name="q5"></a>
-#### 5. How can Zonemaster judge what is right and wrong?
-There is no final judgement of the health of a domain that can be bestowed by
-anyone. The people behind Zonemaster do not claim that the tool is correct in
-every aspect. Sometimes opinions differ. We have done our very best to create a
-default policy for found errors within
-this project. Hopefully this is a good compromise between what is an actual
-potentially dangerous error and what could be merely seen as a notice or warning.
-The added advantage of the tool is that one can add a policy file suited to
-one's necessity to a specified directory and ask the tool to use that policy
-file when running the tests.
-But as with all things as evolving as DNS the situation is most likely
-changing, what is a notice today could be an error tomorrow. If you really think
-we have made a mistake in our judgement please do not hesitate to drop us an email
-at [zonemaster-users@lists.iis.se](mailto:zonemaster-users@lists.iis.se) (moderated
-mailing list) with a link to your test and an explanation why you think it
-shows something that you consider incorrect.
+#### 5. How can Zonemaster distinguish between what is right and wrong?
+The judgement of Zonemaster is primarily based on the DNS standards as defined in [RFCs].
+It also bases its judgement on DNS best practices, which can be more loosely defined.
+All Zonemaster tests are defined in [Test Case Specifications][Implemented Test Cases]
+in which the references to the standard documents for that test case are found.
+
+The descriptions of message levels such as *notice*, *warning* and *error* are found 
+in [Severity Level Definitions].
+
+Sometimes there are different interpretations of the standards or opinions on what is best practice,
+and the Zonemaster team is always open to input.
+If you think we have made a mistake in our judgement please do not hesitate to send us an email
+at [zonemaster-users@lists.iis.se] (moderated mailing list) with a link to your test result
+and an explanation as to why you think it shows something that you consider incorrect.
 
 <a name="q6"></a>
-#### 6. Does Zonemaster handle IPv6?
-Yes, it does. All tests run over IPv4 will also be run over IPv6 if Zonemaster
-is configured to do so.
+#### 6. Does Zonemaster support IPv6?
+Yes. By default Zonemaster will query name servers both over IPv4 and IPv6, unless explicitly
+configured otherwise.
+Such configuration is accessible through the "Options" button.
 
 <a name="q7"></a>
-#### 7. Does Zonemaster handle DNSSEC?
-Yes, if DNSSEC is available for a domain that is tested by Zonemaster, it will be
+#### 7. Does Zonemaster verify DNSSEC?
+Yes. If DNSSEC is available for a domain that is tested by Zonemaster, it will be
 checked automatically.
+Note that you can also explicitly provide Delegation Signers (DS) records using the
+"Options" button.
```

Comment:
I suggest that this is lifted up to be an FAQ question on its own instead of hidden her. It could be added after question 12:
```
#### 13. Can I test the DS records before they are activated?

Yes. Use the "Options" button and there add provide the Delegation Signer (DS) records to be tested.
Zonemaster will then use those in the same way as if they were already added in the parent zone.
```